elixir/apps/api/lib/api/schemas/resource_schema.ex

```diff
@@ -37,7 +37,20 @@ defmodule API.Schemas.Resource do
       description: "POST body for creating a Resource",
       type: :object,
       properties: %{
-        resource: %Schema{anyOf: [Resource.Schema]}
+        resource: %Schema{
+          anyOf: [
+            Resource.Schema,
+            %Schema{
```

Comment:
Is this correct? I'm not familiar with this lib syntax but suspicious part is that I don't see `connections` map key anywhere here.

Reply:
Ah, good catch.  I think I need to put this in a `properties` key.  Will push an update.